WilliamsGarcia/30-08/HackerRank/Sort/StudentComparator.java

```diff
@@ -0,0 +1,18 @@
+package com.bootcamp.daythree.hackerrank.sort;
+
+import java.util.Comparator;
+
+public class StudentComparator implements Comparator<Student> {
```

Comment:
Why Comparator ?
Why not Comparable?
Do you know the difference?

Reply:
I used Comparator because I needed to compare 2 students. But, that was my question and you answered me in class. Ty for this. Now I know because Comparable is better in this case.👊🏻